core/modules/field/src/kind_setgroup.lua

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
Just to sanity check, this would be things like tasks in Makefiles or filters in VS, correct?

Reply:
set group isn't really constrained to any existing model. The idea of the field kind is to simply be a collection of sets that are named. So for `includeDirs` I defined them as `private` and `public`. The same set group field type could be used for `links` if you wanted a set of `static` and `shared` links. Anywhere there's a field where you want categories could be applicable here.

Reply:
Ahh, we may want to consider a different name then. `group` is already taken (or at least it was in v5 and I see no reason to change that API name in v6).

Reply:
Yeah this one is `setgroup` but I can see the potential confusion. I'll see if there's a different name I can use.